qhub/render.py

```diff
@@ -92,21 +94,28 @@ def render_template(output_directory, config_filename, force=False, dry_run=Fals
     )
 
     if new:
-        print("The following files will be created:")
+        table = Table("The following files will be created:", style="deep_sky_blue1")
```

Comment:
I just had a though: it might be useful to display the files either as a table, json, or tree. Similar to Azure, GCP, AWS command where you can select the `--output` format

Reply:
Add this to an existing ENH issue https://github.com/Quansight/qhub/issues/1478